lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -90,17 +109,16 @@ window.runLighthouseForConnection = function(connection, url, options, requested
   // Add url and config to fresh options object.
   const runOptions = Object.assign({}, options, {url, config});
 
-  lighthouseIsRunning = true;
```

Comment:
I'd prefer to keep the explicit `lighthouseIsRunning = true;` inline here and below but call a `updateBadgeUI()` immediately after each.